### PR TITLE
Adding CMYK support

### PR DESCRIPTION
**Pull request by cmonagle**
Code largely taken from https://github.com/bgrins/TinyColor/issues/118.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,6 +19,7 @@
 				"rgb:\t" + tiny.toRgbString(),
 				"hsl:\t" + tiny.toHslString(),
 				"hsv:\t" + tiny.toHsvString(),
+        "cymk:\t" + tiny.toCmykString(),
```
Comment (redlive) [inline index.html]: Isn't supposed to be cmyk:\t instead of cymk:\t?

Comment (cmonagle) [inline index.html]: Thanks for catching that

**Comment by crertel-packlane** [comment]
Is this still alive? Anything we can do to help?

**Comment by cmonagle** [comment]
I think they're moving towards an extension model, so I'm not sure this PR will ever make it in. The typo mentioned above was fixed though.

See: https://github.com/bgrins/TinyColor/issues/135

**Pull request closed without merging**